spec/capabilities.md

```diff
@@ -0,0 +1,316 @@
+# Capabilities
+
+This reference doc contains details about each capability defined in the w3-protocol suite of services. It does not specify the exact RPC semantics for each service, for example, result schemas, possible error conditions, etc. See [services.md](./services.md) for those details. 
+
+## About the definitions
+
+A capability consists of several components:
+
+- The **ability** is the "verb" of the capability, describing the action an agent can perform. For example, `store/add` allows adding CAR files to the store. Abilities are encoded into the `can` field in a UCAN delegation or invocation.
+- The **resource** is the "noun" of the capability, describing something that an agent is trying to perform an action _on_. Resources are URIs and are encoded into the `with` field of a UCAN.
+- The **caveats** are qualifiers that can constrain delegations or parameterize invocations. See below for more details. Caveats are encoded into the `nb` object field of a UCAN.
+
+In the definitions below, we identify capabilities by the ability name, which is used by the service provider to route invocations to the correct handler. The definitions include what kinds of resource URI are acceptable, as well as optional and required caveats that can be included in an invocation.
+
+The caveats are used for two complementary purposes. When used in an invocation, they act as "function parameters" for the remote procedure call, giving the capability provider the context they need to fulfill the request. For example, the `link` caveat in a `store/add` invocation specifies the CID of the CAR to be stored.
+
+When used in a delegation, caveats act as constraints on the values allowed in an invocation. For example, if a `store/add` delegation has a `size` caveat of 10MB, your invocation's `size` caveat must be less than or equal to 10MB.
+
+### Issuer and audience
+
+UCANs have a notion of "issuer" and "audience", represented by the `iss` and `aud` fields.
```

Comment:
Nit: This may be misleading, because in all of the code they are named `issuer` and `audience` while in their raw JWT form they are called `iss` and `aud`. In examples we often type raw form to keep it short, but in all the code it would be in full form.